docs/doxygen/overviews/xrc_format.h

```diff
@@ -2674,14 +2674,15 @@ should be processed on. It is filtered out and ignored on any other platforms.
 
 Possible elemental values are:
 @beginDefList
-@itemdef{ @c win, Windows }
-@itemdef{ @c mac, macOS (or Mac Classic in wxWidgets version supporting it) }
+@itemdef{ @c msw, Windows, preferred platform name }
+@itemdef{ @c win, Windows, alternative synonym }
```

Comment:
FWIW, I think "msw" is strictly worse than "win" as the platform identifier. Unlike "win", it is not universally recognized as an abbrev. for Windows (see http://google.com/search?q=msw; I don't think I ever saw it in non-wx context…). This value is the *platform* ("Windows", "Mac"), not wx *port* ("wxMSW").

Reply:
But we do use MSW a lot and quite consistently. I'm also not completely unsympathetic to people who follow FSF advice not to use "win" as abbreviation of "Windows", even if I don't see any real problem with this myself, and using "msw" allows to completely side-step the issue.